sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/MeterProviderConfiguration.java

```diff
@@ -33,20 +34,25 @@ static void configureMeterProvider(
           metricExporterCustomizer,
       List<Closeable> closeables) {
 
-    // Configure default exemplar filters.
-    String exemplarFilter =
-        config.getString("otel.metrics.exemplar.filter", "trace_based").toLowerCase(Locale.ROOT);
-    switch (exemplarFilter) {
-      case "always_off":
-        SdkMeterProviderUtil.setExemplarFilter(meterProviderBuilder, ExemplarFilter.alwaysOff());
-        break;
-      case "always_on":
-        SdkMeterProviderUtil.setExemplarFilter(meterProviderBuilder, ExemplarFilter.alwaysOn());
-        break;
-      case "trace_based":
-      default:
-        SdkMeterProviderUtil.setExemplarFilter(meterProviderBuilder, ExemplarFilter.traceBased());
-        break;
+    Optional<ExemplarFilter> spiExemplarFilter = spiHelper.loadOptional(ExemplarFilter.class);
```

Comment:
ExemplarFilter is going away as a public interface so let's not do this: https://github.com/open-telemetry/opentelemetry-specification/pull/3820

Reply:
removed